Source/SPDataImport.m

```diff
@@ -782,8 +782,8 @@ - (void)importCSVFile:(NSString *)filename
 	NSMutableArray *parsedRows = [[NSMutableArray alloc] init];
 	NSMutableArray *parsePositions = [[NSMutableArray alloc] init];
 	NSArray *csvRowArray;
-	NSInteger fileChunkMaxLength = 256 * 1024;
-	NSUInteger csvRowsPerQuery = 50;
+	NSInteger fileChunkMaxLength = 512 * 1024;
+	NSUInteger csvRowsPerQuery = 1000;
```

Comment:
This increases the values toooooo much. Afaik Bigger query might be affected in concurrent access and could potentially lead to a deadlock? So if increase, then let's try smaller - like 200 (4x) and not 1000 (20x) :)